aqualinkd.c

```diff
@@ -694,9 +694,7 @@ int main(int argc, char *argv[])
   bool cmdln_debugRS485 = false;
 
   // struct lws_context_creation_info info;
-  // Log only NOTICE messages and above. Debug and info messages
-  // will not be logged to syslog.
-  setlogmask(LOG_UPTO(LOG_NOTICE));
+  setlogmask(LOG_UPTO(LOG_DEBUG));
```

Comment:
When the mask is set to NOTICE if the user enables DEBUG in aqualinkd.conf journaclt will only show the DEBUG logs.  all the other ones are lost.  Opposite the behavior you'd expect

Reply:
I remove all changes related to syslog and just kept the fix for #88 heap-buffer-overflow in action_web_request